src/App.tsx

```diff
@@ -1,26 +1,53 @@
+import { FC, useEffect, useState } from 'react';
+
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
+
+import { getTodos } from './api';
+import { useAppDispatch, useAppSelector } from './app/hooks';
+import { setTodos } from './features/todos';
+import { selectCurrentTodo } from './features/currentTodo';
+
 import { Loader, TodoFilter, TodoList, TodoModal } from './components';
 
-export const App = () => (
-  <>
-    <div className="section">
-      <div className="container">
-        <div className="box">
-          <h1 className="title">Todos:</h1>
+export const App: FC = () => {
+  const dispatch = useAppDispatch();
+  const [isLoading, setIsLoading] = useState(false);
+  const currentTodo = useAppSelector(selectCurrentTodo);
 
-          <div className="block">
-            <TodoFilter />
-          </div>
+  useEffect(() => {
+    setIsLoading(true);
+
+    getTodos()
+      .then(data => {
+        dispatch(setTodos(data));
+      })
+      .finally(() => {
+        setIsLoading(false);
+      });
+  }, [dispatch]);
```

Comment:
The `useEffect` hook has `dispatch` as its only dependency. If `getTodos` or `setTodos` are functions that might change or if there are other dependencies that should be considered, they should be included in the dependency array. This ensures that the effect runs correctly when those dependencies change.